services/migration_service/migration_files/20210202145952_add_runs_idx_ts_epoch_flow_id.sql

```diff
@@ -0,0 +1,60 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+-- Others
+
+CREATE INDEX runs_v3_idx_ts_epoch ON runs_v3 (ts_epoch);
+
+CREATE INDEX runs_v3_idx_gin_tags ON runs_v3 USING gin (tags, system_tags);
```

Comment:
when creating an index add `IF NOT EXISTS` to prevent the migration scripts from getting stuck if a user already manually added the indices

Reply:
good point, forgot about this :ok_hand:

Situation can also happen if metadata service ran the create tables command and created indices along with it, before migration service was used (can and will happen a lot during local development).